nl/firefox-ios.xliff

```diff
@@ -1009,6 +1003,10 @@
         <target>Wilt u het wachtwoord op %@ bijwerken?</target>
         <note>Prompt for updating a password with on username. The parameter is the hostname of the site.</note>
       </trans-unit>
+      <trans-unit id="Don't Enable">
```

Comment:
@st3fan Ask Jean / Vijay if these strings actually need to be in here. My guess is not.